vnext/Microsoft.ReactNative/Views/Image/ReactImage.cpp

```diff
@@ -181,8 +208,7 @@ winrt::fire_and_forget ReactImage::SetBackground(bool fireLoadEndEvent) {
 
       strong_this->m_surfaceLoadedRevoker = surface.LoadCompleted(
           winrt::auto_revoke,
-          [weak_this, compositionBrush, surface, fireLoadEndEvent](
-              winrt::LoadedImageSurface const & /*sender*/,
+          [=](winrt::LoadedImageSurface const & /*sender*/,
```

Comment:
Just left this on another PR, but implicit capture by value can be dangerous (allows unsafe capture of this ptr when accessing members, accidental copies), and is banned by CppCoreGuidelines. We should keep the explicit capture here.

Reply:
The guideline actually seems to have looseded recently. This is the specific wording now. http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f54-if-you-capture-this-capture-all-variables-explicitly-no-default-capture